protos/vehicle_data.proto

```diff
@@ -216,6 +227,188 @@ message LocationValue {
   double longitude = 2;
 }
 
+enum FollowDistanceEnum {
+    FollowDistance1 = 0;
+    FollowDistance2 = 1;
+    FollowDistance3 = 2;
+    FollowDistance4 = 3;
+    FollowDistance5 = 4;
+    FollowDistance6 = 5;
+    FollowDistance7 = 6;
+}
+
+enum ForwardCollisionSensitivityEnum {
+  ForwardCollisionSensitivityOff = 0;
+  ForwardCollisionSensitivityLate = 1;
+  ForwardCollisionSensitivityAverage = 2;
+  ForwardCollisionSensitivityEarly = 3;
+}
+
+enum GuestModeMobileAccessStateEnum {
+    GuestModeMobileAccessStateInit = 0;
```

Comment:
not all vehicles are in `GuestModeMobileAccessStateInit` state? Maybe we should have an unknown state?